src/libasr/codegen/asr_to_cpp.cpp

```diff
@@ -759,7 +760,15 @@ Kokkos::View<T*> from_std_vector(const std::vector<T> &v)
         std::string left_val = src;
         this->visit_expr(*x.m_right);
         std::string right_val = src;
-        src = "(" + left_val + ") + (" + right_val + ")";
+        switch (x.m_op) {
+            case (ASR::stropType::Concat): {
+                src = "std::string(" + left_val + ") + std::string(" + right_val + ")";
+                break;
+            }
+            case (ASR::stropType::Repeat): {
+                // TODO: implement
```

Comment:
There's no such direct idiomatic way to repeat a string in C++ equivalent to the `*` operator we have in Python.
What I did for the implementation of string repetition (Python frontend):
```c++
                std::ostringstream os;
                std::fill_n(std::ostream_iterator<std::string>(os), repeat, std::string(str));
                std::string result_s = os.str();
```
Should we generate something like this now? Any other ideas are appreciated! @certik @Smit-create

Reply:
I think we need a "runtime" library in C++ that this backend then uses. It would be a header file and a cpp file. Some of the Kokkos conversion functions would go there too, that we currently generate at the top of the file.

Reply:
Good idea. Let's do that once we get the basic functionality up and running.